src/Camelot.ViewModels/Implementations/Dialogs/SettingsDialogViewModel.cs

```diff
@@ -33,15 +41,12 @@ public int SelectedIndex
 
         public ICommand CloseCommand { get; }
 
-        public SettingsDialogViewModel(
-            ISettingsViewModel terminalSettingsViewModel)
+        public SettingsDialogViewModel(IList<ISettingsViewModel> terminalSettingsViewModel)
         {
-            TerminalSettingsViewModel = terminalSettingsViewModel;
+            TerminalSettingsViewModel = terminalSettingsViewModel[0];
```

Comment:
`TerminalSettingsViewModel = terminalSettingsViewModel[0];` is not so obvious code